feat: auto-login after admin account setup

laipz8200 opened the pull request:
## Summary
- Implemented automatic login after admin account creation during initial setup
- Users are now directly redirected to the apps page after setup completion
- Eliminates the need to manually login with the same credentials just created

## Related Issue
Fixes #24394

https://github.com/user-attachments/assets/a45322b9-51e6-4692-87a6-610f78e73bdf

## Implementation Details
The solution is implemented entirely in the frontend by:
1. After successful admin account setup, automatically calling the login API with the same credentials
2. Storing the returned access_token and refresh_token in localStorage
3. Redirecting to /apps instead of /signin
4. Includes fallback behavior to redirect to signin if auto-login fails

laipz8200 committed:
feat: auto-login after admin account setup

Automatically authenticate the admin user after account creation during
the initial setup process, providing a seamless onboarding experience.

Fixes #24394

crazywoola commented on web/app/install/installForm.tsx:
```diff
@@ -54,12 +54,32 @@ const InstallForm = () => {
+    if (loginRes.result === 'success') {
+      localStorage.setItem('console_token', loginRes.data.access_token)
+      localStorage.setItem('refresh_token', loginRes.data.refresh_token)
+      router.push('/apps')
```
Prefer replace

https://github.com/langgenius/dify/blob/bb852ef6d20d65c87c4c0eaa5659e21a736c19b9/web/app/signin/normal-form.tsx#L40

laipz8200 committed:
fix: use router.replace instead of router.push

As suggested in PR review, using router.replace prevents users from
navigating back to the setup page after successful authentication
